src/components/enterprise-user-subsidy/enterprise-offers/EnterpriseOffersBalanceAlert.jsx

```diff
@@ -0,0 +1,75 @@
+import React, { useContext } from 'react';
+import classNames from 'classnames';
+import PropTypes from 'prop-types';
+
+import { Container, Alert, MailtoLink } from '@edx/paragon';
+import { WarningFilled } from '@edx/paragon/icons';
+import { sendEnterpriseTrackEvent } from '@edx/frontend-enterprise-utils';
+import { getAuthenticatedUser } from '@edx/frontend-platform/auth';
+import { AppContext } from '@edx/frontend-platform/react';
+import {
+  LOW_BALANCE_CONTACT_ADMIN_TEXT,
+  LOW_BALANCE_ALERT_HEADING,
+  LOW_BALANCE_ALERT_TEXT,
+  NO_BALANCE_CONTACT_ADMIN_TEXT,
+  NO_BALANCE_ALERT_HEADING,
+  NO_BALANCE_ALERT_TEXT,
+  OFFER_BALANCE_CLICK_EVENT,
+} from './data/constants';
+
+const EnterpriseOffersBalanceAlert = ({ hasNoEnterpriseOffersBalance }) => {
+  const {
+    enterpriseConfig: { uuid: enterpriseCustomerUUID, adminUsers },
+  } = useContext(AppContext);
+
+  const adminText = hasNoEnterpriseOffersBalance ? NO_BALANCE_CONTACT_ADMIN_TEXT : LOW_BALANCE_CONTACT_ADMIN_TEXT;
+  const className = hasNoEnterpriseOffersBalance
+    ? 'no-offers-balance-alert-with-cta' : 'low-offers-balance-alert-with-cta';
+  const variant = hasNoEnterpriseOffersBalance ? 'danger' : 'warning';
+  const icon = WarningFilled;
+  const heading = hasNoEnterpriseOffersBalance ? NO_BALANCE_ALERT_HEADING : LOW_BALANCE_ALERT_HEADING;
+  const text = hasNoEnterpriseOffersBalance ? NO_BALANCE_ALERT_TEXT : LOW_BALANCE_ALERT_TEXT;
+
+  const adminEmails = adminUsers.map(user => user.email);
+  const hasAdminEmails = adminEmails.length > 0;
+
+  const { userId } = getAuthenticatedUser();
+  const actions = hasAdminEmails ? [
+    <MailtoLink
+      to={adminEmails}
+      target="_blank"
+      onClick={() => {
+        sendEnterpriseTrackEvent(
+          enterpriseCustomerUUID,
+          OFFER_BALANCE_CLICK_EVENT,
+          {
+            userId,
```

Comment:
Pretty sure the Segment event will already have the `userId` in place even without passing it as an event property, since we make an `identify` call via `@edx/frontend-platform` during initialization of the analytics service: https://github.com/openedx/frontend-platform/blob/992094d1697ad1f949870915498c89a1143f3d6c/src/initialize.js#L171